src/common/fam_internal.h

```diff
@@ -409,19 +414,23 @@ inline void decode_offset(uint64_t offset, int *extentIdx, uint64_t *startPos) {
 inline string protocol_map(string provider) {
     std::map<std::string, int> providertypes;
     string protocol;
-    providertypes["verbs;ofi_rxm"] = 1;
-    providertypes["sockets"] = 2;
-    providertypes["tcp"] = 3;
+    providertypes["cxi"] = 1;
+    providertypes["verbs"] = 2;
```

Comment:
Please check if this has to be "verbs;ofi_rxm" instead of "verbs"

Reply:
Addressed